src/app/screens/Onboard/Onboard/index.tsx

```diff
@@ -0,0 +1,89 @@
+import {
```

Comment:
Why is this file inside Onboard/Onboard (twice?)

Reply:
![image](https://github.com/getAlby/lightning-browser-extension/assets/55848322/f3d5050d-83a1-431b-95e1-259a7b6522ed)
cause that is somehow the file structure is

Reply:
This feels confusing, should it at least be called Onboard/OnboardingPrompt? CC @reneaaron

Reply:
I kind of agree. I'll change the structure a bit during my final review :ok_hand: